parsers/jscript.c

```diff
@@ -2313,21 +2323,17 @@ static bool parseStatement (tokenInfo *const token, bool is_inside_class)
 		{
 			/*
 			 * Creates tags for each of these class methods
-			 *     ValidClassOne.prototype = {
+			 *     objectOne = {
 			 *         'validMethodOne' : function(a,b) {},
 			 *         'validMethodTwo' : function(a,b) {}
 			 *     }
 			 * Or checks if this is a hash variable.
 			 *     var z = {};
 			 */
-			bool anonClass = vStringIsEmpty (name->string);
-			if (anonClass)
-				anonGenerate (name->string, "AnonymousClass", JSTAG_CLASS);
-			has_methods = parseMethods(token, name, false);
-			if (has_methods)
-				makeJsTagCommon (name, JSTAG_CLASS, NULL, NULL, anonClass);
-			else
-			{
+			bool anonObject = vStringIsEmpty (name->string);
+			if (anonObject)
+				anonGenerate (name->string, "anonymousObject", JSTAG_VARIABLE);
+			parseMethods(token, name, false);
```

Comment:
With the change, the value returned from parserMethods are ignored, and the tag for the anonymous object is not made. I think this is not a good idea. Could you keep the original code with just changing the kind like:
```c
			has_methods = parseMethods(token, name, false);
			if (has_methods)
				makeJsTagCommon (name, JSTAG_VARIABLE, NULL, NULL, anonObject);
			else
			{
```
?